integration/cluster_test.go

```diff
@@ -270,6 +270,33 @@ func TestIssue2746(t *testing.T) {
 	clusterMustProgress(t, c.Members)
 }
 
+// Ensure etcd can update attribute of removed member
+func TestIssue2904(t *testing.T) {
+	defer afterTest(t)
+	c := NewCluster(t, 1)
+	c.Launch(t)
+	defer c.Terminate(t)
+
+	c.AddMember(t)
+	// this member is gonna to be removed
```

Comment:
the comment does not make a lot of sense to me. we stop it since we want to restart it later to force it update its attr.